common/national_focus/SRS.txt

```diff
@@ -1737,54 +1876,71 @@ shared_focus = {
 shared_focus = {
 	id = SRS_war_1
 	icon = GFX_goal_stg_socialist_revolution_griffon
+	
+	relative_position_id = SRS_war_tree
 	x = 1
 	y = 1
-	cost = 10
-	relative_position_id = SRS_war_tree
-
+	
 	prerequisite = { focus = SRS_war_tree }
 	
+	cost = 10
+	
 	ai_will_do = {
 		factor = 20
 	}
-
+	
+	bypass = {
+		NTR = {
+			OR = {
+				AND = {
+					exists = yes
+					has_government = communism
+				}
+				
+				all_cores_owned_by_ROOT = yes
+			}
+		}
+	}
+	
 	available = {
 		has_government = communism
 	}
-
+	
 	completion_reward = {
 		log = "[GetDateText]: [Root.GetName]: Focus SRS_war_1"
+		
 		custom_effect_tooltip = SRS_communalist_delegates_hold_a_majority_tt
 		effect_tooltip = { add_timed_idea = { days = 650 idea = SRS_weapons_for_the_front } }
+		
 		custom_effect_tooltip = SRS_internationalist_delegates_hold_a_majority_tt
 		effect_tooltip = { add_timed_idea = { days = 650 idea = SRS_defend_our_homes } }
-		hidden_effect = { 
-			if = {
+		
```

Comment:
@Jas-PY please review these changes

Reply:
There is no need for this, these two checks are mutually exclusive, but for some reason both used IF (such errors are repeated many times, but I don’t want to correct them all).